libs/components/forms/src/lib/modules/file-attachment/file-drop.component.html

```diff
@@ -89,23 +89,30 @@
             }}
           </div>
         </div>
-        <div class="sky-margin-stacked-sm">
+        <div class="sky-margin-stacked-xs">
           <input
             type="text"
             class="sky-form-control"
             [attr.aria-label]="
               linkUploadAriaLabel ||
               ('skyux_file_attachment_file_upload_link_label' | skyLibResources)
             "
-            [attr.placeholder]="
-              'skyux_file_attachment_file_upload_link_placeholder'
-                | skyLibResources
-            "
+            [attr.aria-describedby]="hintText.id"
             [(ngModel)]="linkUrl"
             (blur)="onLinkBlur()"
             (keyup)="addLinkEnter($event)"
           />
         </div>
+        <div
+          skyId
+          #hintText="skyId"
+          class="sky-font-deemphasized sky-file-drop-hint-text sky-margin-stacked-sm"
+        >
+          {{
+            'skyux_file_attachment_file_upload_link_placeholder'
```

Comment:
The screenshots in the story have the hint text being, "Start with http:// or https://"

Reply:
Changed